processor/fanoutconnector.go

```diff
@@ -12,30 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package fanoutprocessor contains implementations of Trace/Metrics processors
-// that fan out to multiple other processors.
-package fanoutprocessor
+package processor
 
 import (
 	"context"
 
 	"github.com/open-telemetry/opentelemetry-service/consumer"
 	"github.com/open-telemetry/opentelemetry-service/consumer/consumerdata"
 	"github.com/open-telemetry/opentelemetry-service/oterr"
-	"github.com/open-telemetry/opentelemetry-service/processor"
 )
 
-// NewMetricsProcessor wraps multiple metrics consumers in a single one.
-func NewMetricsProcessor(mcs []consumer.MetricsConsumer) processor.MetricsProcessor {
-	return metricsConsumers(mcs)
+// This file contains implementations of Trace/Metrics connectors
+// that fan out the data to multiple other consumers.
+
+// NewMetricsFanOutConnector wraps multiple metrics consumers in a single one.
+func NewMetricsFanOutConnector(mcs []consumer.MetricsConsumer) MetricsProcessor {
+	return metricsFanOutConnector(mcs)
 }
 
-type metricsConsumers []consumer.MetricsConsumer
+type metricsFanOutConnector []consumer.MetricsConsumer
 
-var _ processor.MetricsProcessor = (*metricsConsumers)(nil)
+var _ MetricsProcessor = (*metricsFanOutConnector)(nil)
 
 // ConsumeMetricsData exports the MetricsData to all consumers wrapped by the current one.
-func (mcs metricsConsumers) ConsumeMetricsData(ctx context.Context, md consumerdata.MetricsData) error {
+func (mcs metricsFanOutConnector) ConsumeMetricsData(ctx context.Context, md consumerdata.MetricsData) error {
```

Comment:
nit of nitness: `mcs` now doesn't match the type as before, what about `mfc`? 

While at that `mdp` on the loop also doesn't match well (seems to refer to MetricsDataProcessor), `mc` seems better.

Reply:
I'll fix.

Reply:
Fixed.